### PR TITLE
Player form validation

### PR DESCRIPTION
**Pull request by remiriv**
## Proposed changes

---

- Ajout du bouton pour valider un formulaire complètement rempli 
- Ajout de la pop up avec le texte pour la RGPD 

## Types of changes

---

What types of changes does your code introduce to ogre ?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe):

## Checklist

---

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

offseason_ogre/blob/master/CONTRIBUTING.md) doc

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation and modify the README.md (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] If the feature changes current behavior, reasons why your solution is better.

## What is the current behavior?

---

-
-
-

## What is the new behavior?

---

-
-
-

## Further comments

---

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

## Where has this been tested?

---

Docker version: …
Operating System: …
node version: …
postgresql version: …

### CODE REVIEW

File: packages/client/src/modules/administration/Games/Game/GamePlayers.tsx [hunk 1]
```diff
@@ -68,7 +67,7 @@ function GamePlayers({ game }: { game: IGame }): JSX.Element {
     return {
       ...player,
       teamId: currentGame?.team.id || 0,
-      formStatus: currentGame?.profile?.status || "draft",
+      formStatus: currentGame?.profile?.status || "Non rempli",
```
Comment (louis-sanna-dev) [inline packages/client/src/modules/administration/Games/Game/GamePlayers.tsx]: @remiriv pk en français ce status?

File: packages/client/src/modules/play/Personalization/PersonalizationForm.tsx [hunk 2]
```diff
@@ -130,6 +147,7 @@ function PersonalizationForm() {
         userId: user?.id,
         update: {
           ...getNonNullValues(),
+          profileStatus: "draft",
           origin: `player_${user?.id}_${gameId}`,
```
Comment (louis-sanna-dev) [inline packages/client/src/modules/play/Personalization/PersonalizationForm.tsx:151]: @remiriv évitons de dupliquer la logique `player_${user?.id}_${gameId}`.

File: packages/client/src/modules/play/Personalization/common/Dialogs.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            sx={{ border: 1, borderColor: "secondary", mt: 1 }}
+            onClick={() => setBackArrowDialogOpen(false)}
+          >
+            Non
+          </Button>
+          <Button
+            color="primary"
+            variant="contained"
+            sx={{ border: 1, borderColor: "secondary" }}
+            onClick={() => navigate(`/play/games/${gameId}/personalize/choice`)}
```
Comment (louis-sanna-dev) [inline packages/client/src/modules/play/Personalization/common/Dialogs.tsx:72]: @remiriv on peux faire

```js
const navigate = useNavigate();
```
au lieu de le passer en argument.

File: packages/client/src/modules/administration/Games/Game/GamePlayers.tsx (hunk 1 above)
Comment (remiriv) [reply packages/client/src/modules/administration/Games/Game/GamePlayers.tsx]: c'et modifié

**Review by louis-sanna-dev — APPROVED** [review]
@remiriv il y a tjs le bug sur le formulaire qui disparait au reload. Sur staging:

Screenshot 2022-11-18 at 13 54 30
Screenshot 2022-11-18 at 13 54 38

**Comment by remiriv** [comment]
@louis-sanna-eki j'ai fait les modifs pour cette PR

**Pull request merged**